cli/src/commands/edit.rs

```diff
@@ -21,10 +21,10 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Sets the specified revision as the working-copy revision
 ///
-/// Puts the contents of a commit in the working copy for editing. Any changes
-/// you make in the working copy will update (amend) the commit.
+/// Note, it is generally recommended to instead to use `jj new` and `jj amend`. Creating
```

Comment:
nit: I would say that `jj squash` is more idiomatic than `jj amend` (the latter exists for familiarity for git/hg users)

This inspired me to write #3252 :)

Reply:
Also will do

Reply:
I don't think so. I would say that aliases are generally less idiomatic, but `jj op` and `jj st` are perhaps more idiomatic than `jj operation` and `jj status`? I guess when the alias is a completely different word (not an abbreviation), it's generally because of some kind of compatibility.